packages/ckeditor5-table/src/converters/tableproperties.js

```diff
@@ -19,7 +19,7 @@
  * @param {Boolean} [options.reduceBoxSides=false]
  */
 export function upcastStyleToAttribute( conversion, options ) {
-	const { viewElement, defaultValue, modelAttribute, styleName, reduceBoxSides = false } = options;
+	const { viewElement, defaultValue, modelAttribute, styleName, reduceBoxSides = false, skipCondition = () => false } = options;
```

Comment:
Maybe we could find a better name? Also, remember to add this option to the JsDoc above.